packages/strapi-design-system/src/ToggleInput/ToggleInput.tsx

```diff
@@ -19,21 +18,36 @@ const ClearButton = styled(TextButton)`
   margin-left: auto;
 `;
 
+interface ToggleInputProps {
+  checked?: boolean;
+  clearLabel?: string;
+  disabled?: boolean;
+  error?: string;
+  hint?: string;
+  id?: string;
+  label?: string;
+  labelAction?: React.ReactNode;
+  name?: string;
+  onClear?: () => void;
+  required?: boolean;
+  size?: keyof typeof sizes.input;
```

Comment:
We type this a lot manually, Maybe it's worth exporting a union from somewhere central for `InputSizes = "S" | "M"` 🤔  wdyt?

Reply:
yeah that'd be cool, I think it'd be something like `type InputSizes = keyof ThemeSizes['input']` maybe?